system-test/spanner.ts

```diff
@@ -44,6 +44,24 @@ const spanner = new Spanner({
   projectId: process.env.GCLOUD_PROJECT,
   apiEndpoint: process.env.API_ENDPOINT,
 });
+const RETRYOPTIONS: CallOptions = {
```

Comment:
This is pretty trivial, but could this be renamed to `GAX_OPTIONS`? I'm thinking it might be more clear where it's used to understand the type of object we're providing.

Reply:
Sure, done!